dsc/dscCiCd.md

```diff
@@ -0,0 +1,456 @@
+---
+title:   Building a CI-CD pipeline with DSC, Pester, and Visual Studio Team Services
+ms.date:  2017-03-03
+keywords:  powershell,DSC
+description:  
+ms.topic:  article
+author:  eslesar
+manager:  carmonm
+ms.prod:  powershell
+---
+
+# Building a CI-CD pipeline with DSC, Pester, and Visual Studio Team Services
+
+This example demonstrates how to build a Continuous Integration-Continuous Deployment (CI-CD) pipeline by using PowerShell,
+DSC, Pester, and Visual Studio Team Foundation Server (TFS).
+
+After the pipeline is built and configured, you can use it to fully deploy, configure and test a DNS server and associated host records. 
+This process simulates the first part of a pipeline that would be used in a development environment.
+
```

Comment:
I think this is missing a section on why this is a good idea, the benefits associated with it, etc.